static/css/zamboni/stats.less

```diff
@@ -1,4 +1,4 @@
-@import 'lib';
+@import '../impala/lib';
```

Comment:
Ok, so we're moving this file from `/impala/` to `/zamboni/`, but still importing a file from `/impala/lib`. Should we be looking at moving that also into `/zamboni/`, or is that a much bigger task?

Reply:
It's imported in a bunch of stuff still and I wanted to stop before going too far. We can certainly do it later.